src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualWanVpnConfiguration.md

```diff
@@ -0,0 +1,255 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/get-azurermvirtualwanvpnconfiguration
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualWanVpnConfiguration
+
+## SYNOPSIS
+Gets the Vpn configuration for a subset of VpnSites connected to this WAN via VpnConnections. Uploads the generated Vpn
+configuration to a storage blob specified by the customer.
+
+## SYNTAX
+
+### ByVirtualWanObjectByVpnSiteObject (Default)
```

Comment:
Default should always be the interactive parameter set (using names).  Apply everywhere.

Reply:
Fixed